FinniversKit/Sources/Fullscreen/TransactionView/TransactionStepView/TransactionStepContentView/TransactionStepContentView.swift

```diff
@@ -0,0 +1,260 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public protocol TransactionStepContentViewDelegate: AnyObject {
+    func transactionStepContentViewDidTapActionButton(
+        _ view: TransactionStepContentView,
+        inTransactionStep step: Int,
+        withAction action: TransactionStepContentView.ActionButton.Action,
+        withUrl urlString: String?,
+        withFallbackUrl fallbackUrlString: String?
+    )
+}
+
+public class TransactionStepContentView: UIView {
```

Comment:
This is copy+paste from the `TransactionStepView`

Reply:
The `TransactionStepView` is a container and will render two instances of `TransactionStepContent` one for `main` and one for `detail`.